docs/install.md

```diff
@@ -0,0 +1,72 @@
+# Installing BKPR
+
+A Bitnami Kubernetes Production Runtime (BKPR) release consists of a collection of Kubernetes manifests written in Jsonnet plus the accompanying `kubeprod` installer binary. The kubeprod binary deals with all the platform-specific details, evaluates the Jsonnet manifests, and applies them to the existing Kubernetes cluster.
+
+This document walks you through installing the `kubeprod` binary.
+
+## Installing the `kubeprod` installer
+
+The `kubeprod` binary can be installed from pre-built binary releases or be built directly from source.
+
+### Installing from binary releases
+
+BKPR releases are available for 64-bit versions of Linux, macOS and Windows platforms. Download the latest stable version for your platform of choice from the [releases](https://github.com/bitnami/kube-prod-runtime/releases) page.
+
+For convenience lets define the `BKPR_VERSION` environment variable:
+
+```bash
+BKPR_VERSION=$(curl --silent "https://api.github.com/repos/bitnami/kube-prod-runtime/releases/latest" | jq -r '.tag_name')
+```
+
+_The command configures the `BKPR_VERSION` variable with the latest stable version of BKPR. If you wish to use a pre-release or a specific version, please set it up accordingly._
+
+* Step 1: Use the following commands to download the desired release:
```

Comment:
Instead of an unnumbered bulleted list with numbered steps, why not just use a numbered list? Applies to this set of steps as well as the next one.